Example/Example Swift/ComponentAPICoordinator.swift

```diff
@@ -409,15 +409,25 @@ extension ComponentAPICoordinator: CameraViewControllerDelegate {
         validate([document]) { result in
             switch result {
             case .success(let documentRequests):
-                self.documentRequests.append(contentsOf: documentRequests)
-                self.upload(documentRequests: documentRequests)
-                
-                if self.giniConfiguration.multipageEnabled, self.documentRequests.count > 1 {
-                    if let imageDocument = document as? GiniImageDocument {
+                switch document {
```

Comment:
I'm not sure a switch is appropriate here. It's just a special case for a QR code. Everything else is the same (I assume a PDF document cannot appear here). If we add another type, it would fall through to the break statement. That's why I think a simple if statement would be better.

Reply:
You are right, and `ifelse` would be better here.